admissions/utils.py

```diff
@@ -407,11 +410,179 @@ def internal_group_applicant_data(internal_group):
     ).first()
     positions_to_fill = data.available_positions
 
+    # This now counts people who did not interview, we should probably have a
+    # purge step when moving from open admission to discussion
+    # maybe have this in its own mutation
+    """
+    Admission 
+    Configure -> Open -> Discussing -> Locked -> Closed
+    This should be a irreversible process and each step should imply some transition handling
+    If we go from configure to open we create interview objects
+    If we go from Open to Discussing we should delete all data on people who did not interview
+    If we go from Discussing to Locked we throw error if we have not evaluated everyone
+    If we go from locked to closed we create user accounts
+    Should probably not send an email. When we call and say yes this should probably be its own table?
+    I am missing some intermediary stage here
+    """
+    evaluated = all_priorities.filter(internal_group_priority__isnull=False).count()
+    current_progress = int(math.ceil((evaluated / positions_to_fill) * 100))
+
     return InternalGroupApplicantsData(
         internal_group=internal_group,
         first_priorities=first_priorities,
         second_priorities=second_priorities,
         third_priorities=third_priorities,
-        current_progress=want_count,
+        current_progress=current_progress,
         positions_to_fill=positions_to_fill,
     )
+
+
+def priority_to_number(internal_group_position_priority_option):
+    if internal_group_position_priority_option == InternalGroupStatus.WANT:
+        return 100
+    elif internal_group_position_priority_option == InternalGroupStatus.PROBABLY_WANT:
+        return 90
+    elif internal_group_position_priority_option == InternalGroupStatus.RESERVE:
+        return 80
+    elif internal_group_position_priority_option == InternalGroupStatus.INTERESTED:
+        return 70
+    else:
+        return 0
```

Comment:
I think this is probably redundant. At the end we will only sit with WANT and RESERVE? Check this with someone